programs/test/benchmark.c

```diff
@@ -668,7 +772,13 @@ int main( int argc, char *argv[] )
             mbedtls_snprintf( title, sizeof( title ), "RSA-%d", keysize );
 
             mbedtls_rsa_init( &rsa, MBEDTLS_RSA_PKCS_V15, 0 );
```

Comment:
Can `mbedtls_rsa_init` fail with `FEATURE_UNSUPPORTED` or any other errors?

Reply:
perhaps, but `rsa_init()` returns void, and according to the coding standards:
>The `init()` function must always return `void`

Reply:
OK